.github/CONTRIBUTING.md

```diff
@@ -37,13 +37,14 @@ This means you care about the quality of your contributions.
     * When adding a new aport: testing/apkname: new aport
     * When modifying an aport: testing/apkname: short description about changes
   * If needed provide a proper formatted (line wrapped) description of what your patch will do. You can provide a description in the PR, but you must include a message for this specific commit in the commit description. If in the future we would like to distance ourselves from Github the PR information could be lost.
-5. Open your copy of the aports repository at github.com and switch to your feature branch. You should now see an option to create your PR. [More info](https://help.github.com/articles/creating-a-pull-request/)
-6. Wait for an Alpine Linux developer to review your changes.
-7. If all is ok your PR will be merged but if a developer asks for changes please do as follows:
+5. Add your branch to your copy of the aports repository on GitHub `git push -u origin my_new_feature`.
```

Comment:
why not name the canonical upstream remote `origin` and the fork remote your `<username>` at github? Then fetch origin to get latest changes and always push to the remote named as your username on github?

Remotes in my case:
origin: https://github.com/alpinelinux/aports
HRio: https://github.com/HRio/aports

I only fetch `origin` and push to `HRio`.

New features I create like this `git fetch origin; git checkout -b <topic> origin/master` (or for backport: git checkout 310-<topic> origin/3.10-stable`

or do others name the canonical upstream something else than origin? like `upstream`? I feel it a bit confusing to name my forks `origin`.

Reply:
Huh. Honestly I don't know; the steps in this patch are what I was given on IRC when I tried to submit a patch and it didn't work. I am 100% open to better workflows; my only goal here is to make it so that a beginner who's not previously used the Alpine process for contributions can follow the provided instructions successfully.